process/development_practices.md

```diff
@@ -23,4 +23,13 @@ When a new developer joins our team, we'll often find they need some guidance on
     * Don't be scared to speak to the customer.
     * Talk to the UI designers if unsure about anything visual, behaviour etc.
 * Ask somebody for help if you've run into a problem and can't solve it within ~1hr (it's pretty likely somebody else has encountered it)
-* Client showcases are really important. You should always try to attend these.
\ No newline at end of file
+* Client showcases are really important. You should always try to attend these.
+
+
+## Red flags on projects
+
+This is a list of red flags to review every day to improve project delivery:
+
+- [ ] There is no showcase currently planned for the current piece of work
+- [ ] The current epic is longer than a week (it should be broken down in multiple smaller ones)
+- [ ] Customer does not know when the current epic is expected to be finished
```

Comment:
* Huge commits
* Production deploy > 1 day ago
* Not being clear on who the customer is